ui/lib/apps/Statement/pages/StatementsOverviewPage.tsx

```diff
@@ -50,33 +44,21 @@ export default function StatementsOverviewPage({
       .then((res) => res.data)
   }
 
-  function queryStatementStatus() {
-    return fakeReq('ok')
+  const queryConfig = () => {
+    return dashboardClient.statementsConfigGet().then((res) => res.data)
   }
 
-  function updateStatementStatus() {
-    return fakeReq('ok')
+  const updateConfig = (_instanceId: string, config: StatementConfig) => {
+    return dashboardClient.statementsConfigPost(config).then((res) => res.data)
   }
 
-  const queryConfig = () =>
-    fakeReq({
-      refresh_interval: 100,
-      keep_duration: 100,
-      max_sql_count: 1000,
-      max_sql_length: 100,
-    } as StatementConfig)
-
-  const updateConfig = () => fakeReq('ok')
-
   return (
     <StatementsOverview
       onFetchInstances={queryInstance}
       onFetchSchemas={querySchemas}
       onFetchTimeRanges={queryTimeRanges}
       onFetchStmtTypes={queryStmtTypes}
       onFetchStatements={queryStatements}
-      onGetStatementStatus={queryStatementStatus}
-      onSetStatementStatus={updateStatementStatus}
       onFetchConfig={queryConfig}
```

Comment:
looks like we don't need to repeat these functions again. It's fine to call api client directly in the `<overview>`?

Reply:
Yep, will tidy them and remove all unnecessary codes in the next PR.